MIDI/mathphreak_Learningbaby.jsfx

```diff
@@ -1,15 +1,19 @@
-version: 1.0
+version: 1.0.1
 desc: Learningbaby
 author: Matt Horn
 about:
     # Learningbaby
     The Learningbaby is a Sequencer Baby v2 that learns.
     It will add incoming MIDI notes to the pattern.
-    Changes to SeqBabyv2 I had to make for anything to make sense:
+    
+    Video demonstration/tutorial [here](https://youtu.be/dMTiWafJFmg).
```

Comment:
Adding a [`link`](https://github.com/cfillion/reapack-index/wiki/Packaging-Documentation#link) tag to point to at least the forum thread and youtube tutorial like this could be nice too:

```
links:
    Forum Thread http://forum.cockos.com/showthread.php?p=1845408
    Video demonstration/tutorial https://youtu.be/dMTiWafJFmg
```

![Screenshot](https://i.imgur.com/VwgQE2n.png)